### PR TITLE
[#299] feat(wrappers): add Vampire/setup-wsl@v1

### LINKED ISSUE #299: [Action wrapper(s) request] Vampire/setup-wsl
### Are you sure the action(s) you want aren't already supported?
_Please check https://krzema12.github.io/github-actions-kotlin-dsl/supported-actions/_

Yes

### What action(s) do you need?

https://github.com/Vampire/setup-wsl

### PR DESCRIPTION
**Pull request by Vampire**
closes #299

### CODE REVIEW

File: wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/TextTransformation.kt [hunk 1]
```diff
@@ -10,7 +10,7 @@ fun String.toPascalCase(): String {
     val hasOnlyUppercases = none { it in 'a'..'z' }
     val normalizedString = if (hasOnlyUppercases) lowercase() else this
     return normalizedString.replace("+", "-plus-")
-        .split("-", "_", " ")
+        .split("-", "_", " ", ".")
```
Comment (krzema12) [inline wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/TextTransformation.kt:13]: I'd appreciate adding a test case to

https://github.com/krzema12/github-actions-kotlin-dsl/blob/529c89932adff7a7d67e31f22d5e73f444d58e17/wrapper-generator/src/test/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/TextTransformationTest.kt#L18

**Review by krzema12 — CHANGES_REQUESTED** [review]
.

File: wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/TextTransformation.kt (hunk 1 above)
Comment (Vampire) [reply wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/TextTransformation.kt:13]: Definitely, added. :-)

**Comment by Vampire** [comment]
> ktlint fails :)

Yeah, strange IntelliJ didn't fix it up properly, fixed now.

**Review by krzema12 — APPROVED** [review]
:tada:

**Pull request merged**